src/common/options/crimson.yaml.in

```diff
@@ -65,3 +65,13 @@ options:
   level: dev
   desc: The record fullness threshold to flush a journal batch
   default: 0.95
+- name: seastore_default_object_data_reservation
```

Comment:
seastore_default_max_object_size

max_object_size is more meaningful to a user.